packages/atomic/src/components/facets-v1/facet-value-checkbox-rating/facet-value-checkbox-rating.tsx

```diff
@@ -0,0 +1,48 @@
+import {FunctionalComponent, h} from '@stencil/core';
+import {randomID} from '../../../utils/utils';
+import {FacetValueProps} from '../facet-common';
+
+export const FacetValueCheckboxRating: FunctionalComponent<FacetValueProps> = (
+  props,
+  children
+) => {
+  const id = randomID('facet-value-');
+  const count = props.numberOfResults.toLocaleString(props.i18n.language);
+  const ariaLabel = props.i18n.t('facetValue', {
+    value: props.displayValue,
+    count: props.numberOfResults,
+  });
+
+  return (
+    <li key={props.displayValue} class="flex items-center">
+      <button
```

Comment:
See if you could extract the common parts between FacetValueCheckbox and this one, same for the link.
For example, the `button` seems identical, others element reuse the same tailwind classes. Maybe you can leverage `@apply` in the common `.pcss` file.